locales/zh-hans.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Page": "",
+  "of": " / ",
```

Comment:
It would be great if you can provide a good translation for this text, maybe for you it works fine but for most of the users will not. You can submit another translation in this PR and overwrite it just for you on your own Ghost instance ;)

Reply:
In Chinese，we use quantifier after a number. For example:
Page 1 --- 第一页
Page 1 of 3 ---- 第一页, 共3页

Thus, it is hard to deal with "页" in this situation. A better solution is:
"Page % of %": "第一页, 共3页"
It looks like the template need to modify.

Whatever, it is ok of Chinese to understand "1/3" :)

Reply:
Ok then, thanks!